content/knowledge-base/guides/learn_more.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Learn More
+weight: 307
+---
```

Comment:
Nit - I think we mostly use hyphens in filenames. Would be nice to not use an underscore in this one (can change in future).

Reply:
This isn't from a change, there are multiple legacy files that use underscores. This particular page comes from `/content/<version>/reference/learn_more.md`